### PR TITLE
LIT6730 to 37

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
IDs for texts from Abbadie 134 and small revision in LIT5596.

### CODE REVIEW

**Review by thea-m — APPROVED** [review]
I would always say "the Devil", but that might be a Germanism :) @eu-genia ?

File: 5001-6000/LIT5696MagicPrayer.xml [hunk 1]
```diff
@@ -4,8 +4,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ሐጹረ፡ መስቀል፡</title>
-                <title xml:lang="gez" type="normalized" corresp="#t1">Ḥaṣura Masqal</title>
+                <title xml:lang="en" xml:id="t1">Rampart of the Cross (General record)</title>
+                <title xml:lang="gez" xml:id="t2">ሐጹረ፡ መስቀል፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Ḥaṣura Masqal</title>
+
```
Comment (eu-genia) [inline 5001-6000/LIT5696MagicPrayer.xml:10]: I think an abstract would be helpful for clarity and disambiguation, here and elsewhere, as to see the relations (or incipits) one must go to another page, an abstract is what one sees at once

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by eu-genia** [comment]
> I would always say "the Devil", but that might be a Germanism :) @eu-genia ?

of course it should be **the** Devil, with an article, it is a common noun, not a personal name ))

File: 5001-6000/LIT5696MagicPrayer.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5696MagicPrayer.xml]: I added a short clarification. As I understnd now, there is probably no "standard" main stream tradition of Hasura masqal prayer, or at least it has not been clearly identified yet. Various protective texts referring to Cross and its protection may be called Hasura masqal prayers (also if the words Hasura masqal does not occur there). It cannot be excluded, of course, that a part of those texts build up one tradition (of a specific work called Hasura masqal). So it is a bit vague and we just have to collect further material.

File: 5001-6000/LIT5696MagicPrayer.xml [hunk 2]
```diff
@@ -37,7 +37,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p><foreign xml:lang="gez">Ḥaṣura masqal</foreign> is a generic title for protective prayers that
+                  refer, in one way or another, to the Holy Cross and its apotrophaic function (with or without the use of
```
Comment (eu-genia) [inline 5001-6000/LIT5696MagicPrayer.xml:41]: apotropaic

File: 5001-6000/LIT5696MagicPrayer.xml [hunk 3]
```diff
@@ -37,7 +37,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <profileDesc>
             <creation/>
             <abstract>
-                <p/>
+                <p><foreign xml:lang="gez">Ḥaṣura masqal</foreign> is a generic title for protective prayers that
```
Comment (eu-genia) [inline 5001-6000/LIT5696MagicPrayer.xml:40]: mark up `<ref type="authFile" target="TrueCross">Holy Cross</ref>` and/or add relation
`<relation name="ecrm:P129_is_about" active="LIT5696MagicPrayer" passive="TrueCross">`

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: 5001-6000/LIT5696MagicPrayer.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5696MagicPrayer.xml:41]: done

File: 5001-6000/LIT5696MagicPrayer.xml [hunk 4]
```diff
@@ -37,7 +37,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  refer, in one way or another, to the Holy Cross and its apotrophaic function (with or without the use of
+                  <foreign>ʾasmāt</foreign>).</p>
             </abstract>
             <textClass>
                 <keywords>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5696MagicPrayer.xml:45]: done

File: 5001-6000/LIT5696MagicPrayer.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply 5001-6000/LIT5696MagicPrayer.xml:40]: done

**Review by eu-genia — APPROVED** [review]

**Pull request merged**